src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -55,13 +53,69 @@ void createStation() {
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @Test
+    @DisplayName("등록된 지하철역 목록을 조회한다.")
+    void showStations() {
+        // given
+        String stationOne = "서울대입구역";
+        String stationTwo = "봉천역";
+        createStation(stationOne);
+        createStation(stationTwo);
```

Comment:
```suggestion
        Stations stations = StationSteps.지하철역_생성(Arrays.asList(MockStations.서울대입구역, MockStations.봉천역));
```

이런식으로 만들 수도 있지 않을까 싶어요 ㅎㅎ